### ISSUE TITLE
apparently in some cases tempo returns None as worklogid

### ISSUE THREAD

**Issue by fizyk** [ClearcodeHQ/jira_timemachine#22]
### What action do you want to perform

Migrate worklogs to from source jira to destination

### What are the results

Fails on exception when reading data from destiantion jira (no worklog id)

### What are the expected results

Dat ais migrated. Apparently we've got either jiraWorklogid OR tempoWorklogID we could require one or the other. Plus, destination ought to have all required data for confirmation within worklog message.

**Comment by fizyk** [comment]
trying to update the issue seems to also solve the problem though ;)

**Comment by mmaslowskicc** [comment]
So one of the IDs is initially missing, but a following request gets both? We cannot choose the other ID for this reason, since it will duplicate worklogs on a rerun.

I don't see any other solution than skipping the worklog with a warning and optionally retrying.

**Comment by fizyk** [comment]
the issue was with the additional worklog in desitination jira. Apparently tempo stored the worklog but I think it failed to push it to the jira itself, hence the lack of the jiraWorklogID. while fixing, I updated the worklog, and the jiraWorklogID appeared.

I think that if we run sync through tempo, we can safely require the tempo id ( https://tempo-io.github.io/tempo-api-docs/#worklogs ) the tempoWorklogId is required. Jira's is not.

**Comment by mmaslowskicc** [comment]
Yes: if only jiraWorklogID is missing and it's not currently used, then it's ok to not require it.

**Comment by mmaslowskicc** [comment]
When I observed this issue, the Jira ID did not reappear, but I always synchronize only the current day. My current workaround:
```
diff --git a/src/jira_timemachine/__init__.py b/src/jira_timemachine/__init__.py
index 3c0bcc3..29d3f64 100644
--- a/src/jira_timemachine/__init__.py
+++ b/src/jira_timemachine/__init__.py
@@ -106,7 +106,7 @@ class TempoClient(object):
                     continue
                 try:
                     yield Worklog(
-                        id=int(row['jiraWorklogId']),
+                        id=int(row['jiraWorklogId'] or row['tempoWorklogId']),
                         tempo_id=int(row['tempoWorklogId']),
                         author=row['author']['accountId'],
                         started=arrow.get('{startDate} {startTime}'.format(**row)),

```

Since there is a potential case of worklog duplication, I think this might need an option for the user to choose which one to use.
